wallet/wallet-rpc-lib/src/cmdline.rs

```diff
@@ -36,6 +36,7 @@ use crate::config::{WalletRpcConfig, WalletServiceConfig};
     group(
         clap::ArgGroup::new("rpc_auth")
             .args(["rpc_cookie_file", "rpc_username", "rpc_password", "rpc_no_authentication"])
+            .multiple(true)
             .required(true)
     ),
 )]
```

Comment:
FYI: without `multiple(true)`, only one of the `rpc_cookie_file`, `rpc_username`, `rpc_password`, `rpc_no_authentication` options can be specified at the same time (e.g. it's impossible to specify both `rpc_username` and `rpc_password`)